File: src/main/java/org/jukeboxmc/form/element/Label.java
package org.jukeboxmc.form.element;

import org.jetbrains.annotations.NotNull;
import org.json.simple.JSONObject;

/**
 * @author GoMint
 * @version 1.0
 */
public class Label extends Element {

    public Label( String id, String text ) {
        super( id, text );
    }

    @Override
    public @NotNull JSONObject toJSON() {
        JSONObject obj = super.toJSON();
        obj.put( "type", "label" );
        return obj;
    }

}

File: JukeboxMC-API/src/main/kotlin/org/jukeboxmc/api/form/element/Label.kt
package org.jukeboxmc.api.form.element

import com.google.gson.JsonObject

/**
 * @author Kaooot
 * @version 1.0
 */
class Label(id: String, text: String) : Element(id,text) {

    override fun toJSON(): JsonObject {
        val jsonObject = super.toJSON()
        jsonObject.addProperty("type", "label")
        return jsonObject
    }
}